langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -150,6 +197,12 @@ public isolated function toHexString(int n) returns string = @java:Method {
 # No `0x` or `0X` prefix is allowed.
 # Returns an error if the parameter `s` is not in an allowed format.
 #
+# ```ballerina
+# int:fromHexString("1A5F") ⇒ 6751
+# 
+# int:fromHexString("1Y4K") ⇒ error
```

Comment:
Shall we add an example of converting a hex value resulting in a minus integer?

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/38917/commits/bef81eacf094807ffddb1ce9a5eb594b4b1b8bee